presto-orc/src/test/java/com/facebook/presto/orc/BenchmarkSelectiveStreamReaders.java

```diff
@@ -442,11 +443,11 @@ private final Object createValue(Type type, float filterRate)
         {
             switch (withNulls) {
                 case NONE:
-                    return new Pair<>(false, 1 - filterRate);
+                    return new ImmutablePair<>(false, 1 - filterRate);
```

Comment:
ImmutablePair.of(x, y) - https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/tuple/ImmutablePair.html#of-L-R-